sync/linux/controlplane.sh

```diff
@@ -141,6 +141,80 @@ EOF
 kubeadm token create --print-join-command >> /var/sync/shared/kubejoin.ps1
 
 sed -i 's#--token#--cri-socket "npipe:////./pipe/containerd-containerd" --token#g' /var/sync/shared/kubejoin.ps1
+
+cat << EOF > kube-proxy-and-antrea.yaml
```

Comment:
Not sure if this is a space thing, looks like this code already exists?

Reply:
the RBAC was removed in the other PR, but still required for Antrea, letting here for now.